app/models/location.rb

```diff
@@ -108,6 +108,10 @@ def reactivating_soon?
     status == :reactivating_soon
   end
 
+  def deactivates_in_a_long_time?
```

Comment:
For me, at first glance I may interpret "deactivates in a long" time to mean that it's not deactivating soon, instead it is deactivating way after 6 months.